docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -0,0 +1,170 @@
+---
+id: zeebe-backup-and-restore
+title: "Zeebe Backups"
+description: "A guide to creating backup of a running Zeebe cluster."
+keywords: ["backup", "backups"]
+---
+
+:::note
+The API is subject to change.
+:::
```

Comment:
I'm a bit confused about this disclaimer. Is this an early access or preview API?

Reply:
@akeller We do not want to tag it as a "beta" because we will provide production support for this feature. But the API itself is not final, as in it may change in the future. Looping in @felix-mueller